recipes/openpmd-viewer/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openpmd-viewer" %}
+{% set version = "0.9.1" %}
+{% set build = 0 %}
+{% set sha256 = "ac3b2fd00b82fc4ecb2ac7ae991d09fdef3068b0c5e54a08d832c2b7ed65fe6b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-viewer/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - cython
```

Comment:
These should all go into build. But you probably need a compiler here?

Reply:
We currently pre-compile a `.pyx` file on install: https://github.com/openPMD/openPMD-viewer/blob/0.9.1/setup.py#L27-L31
Isn't this the right location for this?

Reply:
You mean host I think

Reply:
So I will move only `setuptools` down and keep `cython` and `numpy` in `build`

Reply:
No, all dependencies listed here should go into `host`. `build` is for compilers & build-tools, `host` for build-time dependencies.

Reply:
Got ya, thanks a lot!